src/overlays/actors/ovl_Bg_Haka_Tubo/z_bg_haka_tubo.c

```diff
@@ -31,18 +29,196 @@ const ActorInit Bg_Haka_Tubo_InitVars = {
     (ActorFunc)BgHakaTubo_Update,
     (ActorFunc)BgHakaTubo_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Haka_Tubo/BgHakaTubo_Init.s")
+ColliderCylinderInit D_80881B60 = {
```

Comment:
could these be named? At the very least, the init chain I think should be static and named sInitChain. But I think these collider cylinder inits can be named too, based on how they're used.